identity.go

```diff
@@ -31,3 +31,15 @@ func (c *IdentityController) List(ctx *app.ListIdentityContext) error {
 		return ctx.OK(result)
 	})
 }
+
+// Show runs the show action.
+func (c *IdentityController) Show(ctx *app.ShowIdentityContext) error {
```

Comment:
There is already one `user` resource, can I renamed that to `authenticated-user`?